.github/workflows/ci-pr-validation.yml

```
@@ -126,70 +293,80 @@ jobs:
-    env:
-      TRIVY_CACHE_DIR: ${{ github.workspace }}/.trivy-cache
     steps:
-      - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd #v6.0.2
+      - name: Checkout
+        uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
         with:
           fetch-depth: 0
-      - name: Governance Drift Check (docs/workflow refs)
+
+      - name: Governance drift check
         if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)